### PR TITLE
New Bot training (audio branch)

### PR DESCRIPTION
**Pull request by jakepoz**

### CODE REVIEW

File: bot_env.py [hunk 1]
```diff
@@ -3,15 +3,16 @@
 from math import inf, ceil
 from typing import Optional
 
-PAN_LOW = 350
-PAN_HIGH = 700
-TILT_LOW = 475
-TILT_HIGH = 725
+PAN_LOW = -35
```
Comment (lostmsu) [inline bot_env.py:6]: Can we keep the old robot code? I still want to combine old data with new data.

Comment (lostmsu) [reply bot_env.py:6]: Conversation note: we could make a separate goal to predict other robot telemetry from video stream only. Might be handy to train on random Internet videos.

File: bot_env.py [hunk 2]
```diff
@@ -3,15 +3,16 @@
...
+PAN_HIGH = 35
+TILT_LOW = -35
+TILT_HIGH = 35
 
 
-def _normalize(val, low, high):
+def normalize_output(val, low, high):
```
Comment (lostmsu) [inline bot_env.py:12]: Isn't it customary to name private module memembers to start with underscore?

File: train.py [hunk 3]
```diff
@@ -230,17 +136,147 @@ def write_bag_cache(bag_file: str, bag_cache_path: str, backbone_onnx_path: str,
...
+        except TimedEntryNotFound:
+            continue
+
+        if not last_reward_button_connection[0]:
+            print(f"Skipping entry {ts} because reward button app was not connected")
+            continue
+
+        final_reward = yolo_reward_value * opt.base_reward_scale
+
+        move_penalty = abs(next_head_cmd).mean() * 0.002
```
Comment (lostmsu) [inline train.py:222]: This is not really a move penalty. For that we'd need to minimize `abs(next_head_cmd - curr_head_cmd)`.

Although it makes sense to make robot prefer looking straight so maybe keep that penalty too.

**Comment by jakepoz** [comment]
@lostmsu Ready to merge in that case?

**Pull request merged**